### PR TITLE
Write the takedown contest template for the EU

### LINKED ISSUE #167: DMCA letter template for EU
See: https://github.com/iv-org/documentation/pull/165

### PR DESCRIPTION
**Pull request by TheFrenchGhosty**
Follow up to the recent takedown requests that multiple EU instances received.

Closes #167

### CODE REVIEW

File: Takedowns.md [hunk 1]
```diff
@@ -28,5 +29,15 @@ Thank you,
 ## Sample Template (EU)
 
 ```
-To Do
+Hello,
+
+I am writing on behalf of {{website}}. I am the webmaster of {{website}} and all of its subdomains.
+
+I see you are filing a claim for {{invidious instance url}}. {{invidious instance url}} hosts Invidious, a private YouTube front-end, meaning it is simply a proxy to access YouTube assets and user generated content without tracking from Google. Thus, all content is mirrored from YouTube and is not stored on our servers and if YouTube chooses to remove an asset, it will no longer appear on our site.
```
Comment (SamantazFox) [inline Takedowns.md]: `all content is proxied*`. Mirrored implies we keep a copy of it.

Also, I'd rephrase the last sentence (from "and if Youtube") like so:
`As such, all takedown requests must be sent to Youtube, as we don't have any control on the data that they ahave on their servers.`

File: Takedowns.md [hunk 2]
```diff
@@ -17,7 +17,8 @@ Hello,
...
 
-I see you are filing a claim for {{invidious instance url}}. {{invidious instance url}} hosts Invidious, a private YouTube front-end, meaning it is simply a proxy to access YouTube assets and user generated content without tracking from Google. Thus, all content is mirrored from YouTube and is not stored on our servers and if YouTube chooses to remove an asset, it will no longer appear on our site. Under United States copyright law 17 U.S.C. § 512(a), part of the Digital Millennium Copyright Act, we cannot be liable for content that "is transmitted through the system or network without modification of its content." It should be noted our  server hosting {{invidious instance url}} is hosted in the United States.
+I see you are filing a claim for {{invidious instance url}}. {{invidious instance url}} hosts Invidious, a private YouTube front-end, meaning it is simply a proxy to access YouTube assets and user generated content without tracking from Google. Thus, all content is mirrored from YouTube and is not stored on our servers and if YouTube chooses to remove an asset, it will no longer appear on our site.
```
Comment (Perflyst) [inline Takedowns.md:20]: > mirrored

Can we change that to "proxied"?

Comment (TheFrenchGhosty) [reply Takedowns.md:20]: True!

File: Takedowns.md [hunk 3]
```diff
@@ -28,5 +29,15 @@ Thank you,
 ## Sample Template (EU)
 
 ```
-To Do
+Hello,
+
+I am writing on behalf of {{website}}. I am the webmaster of {{website}} and all of its subdomains.
+
+I see you are filing a claim for {{invidious instance url}}. {{invidious instance url}} hosts Invidious, a private YouTube front-end, meaning it is simply a proxy to access YouTube assets and user generated content without tracking from Google. Thus, all content is mirrored from YouTube and is not stored on our servers and if YouTube chooses to remove an asset, it will no longer appear on our site.
+Under the directive 2001/29/EC of the European Parliament and of the Council of 22 May 2001 on the harmonisation of certain aspects of copyright and related rights in the information society, "temporary acts of reproduction referred to in Article 2, which are transient or incidental [and] an integral and essential part of a technological process and whose sole purpose is to enable: a transmission in a network between third parties by an intermediary" https://wipolex.wipo.int/en/text/126977 is allowed. It should be noted our {{server hosting}} {{invidious instance url}} is hosted in the European Union.
```
Comment (SamantazFox) [inline Takedowns.md:37]: -> https://wipolex.wipo.int/en/text/126977
I don't think that the link is required. Citing the directive number and chapter+article is enough. If you still want to keep the link, then better point to the official source: https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A02001L0029-20190606

Comment (SamantazFox) [reply Takedowns.md:37]: Also, "It should be noted ... In the EU" should be first in the paragraph: "Our server is in the EU, so  EU law applies. Under EU directive, blablabla..."

File: Takedowns.md [hunk 4]
```diff
@@ -28,5 +29,15 @@ Thank you,
...
+
+In the meantime, while we cannot control the content on YouTube that our site proxies, we recommend that you send a claim to YouTube.
```
Comment (SamantazFox) [inline Takedowns.md:39]: Also, this would be redunddant with if you apply my 2nd recommendation on §1.

**Review by SamantazFox — CHANGES_REQUESTED** [review]

File: Takedowns.md (hunk 1 above)
Comment (SamantazFox) [reply Takedowns.md]: (Ah, forget point 1, already fixed ^^)

Comment (TheFrenchGhosty) [reply Takedowns.md]: Done

File: Takedowns.md (hunk 2 above)
Comment (TheFrenchGhosty) [reply Takedowns.md]: Done

File: Takedowns.md (hunk 3 above)
Comment (TheFrenchGhosty) [reply Takedowns.md]: Done

File: Takedowns.md (hunk 4 above)
Comment (TheFrenchGhosty) [reply Takedowns.md]: See my comment about it in Matrix.

**Comment by SamantazFox** [comment]
I'd personally just add another newline between the two large paragraphs to make the text easier to read, but that's about it. Otherwise it looks good to me!

**Comment by TheFrenchGhosty** [comment]
@SamantazFox Done.
